src/transforms/sort.jl

```diff
@@ -0,0 +1,74 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    Sort(col₁, col₂, ..., colₙ; alg=DEFAULT_UNSTABLE, lt=isless, by=identity, rev=false, order=Forward)
+    Sort([col₁, col₂, ..., colₙ]; alg=DEFAULT_UNSTABLE, lt=isless, by=identity, rev=false, order=Forward)
+    Sort((col₁, col₂, ..., colₙ); alg=DEFAULT_UNSTABLE, lt=isless, by=identity, rev=false, order=Forward)
```

Comment:
We just need to write `kwargs` in the docstring and mention that they are forwarded to `sortperm`. No need to hard-coding the options here. `sortperm` may support more options in the future.